index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <title>Portfolio: setup and mobile version skeleton</title>
+</head>
+
+<body>
+
+    <header id="header">
+
+     <div class="nav-bar">
+        <a href="#" class="hello"> <span class="hello2">Hell</span>o</a>
+        <a href="#" class="portfolio">Portfolio</a>
+        <a href="#" class="about">About</a>
+        <a href="#" class="contact">Contact</a>
+
+        <nav class="button-b">
+            <div class="burger-button"></div>
+        </nav>
+     </div>
+
+    </header>
+
+    <section id="body">
+        <div class="container">
+
+            <div class="left">
+                <h1 class="title">Abel</h1>
+                <h1 class="title">Aguilar</h1>
```

Comment:
- [ ] Good job on implementing other requested changes but as previous reviewer said kindly consider to use the h1 tag only one time for example in this situation you can put Abel Aguilar inside of only one h1. Keep rocking 💪